rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -933,6 +933,7 @@ public JobId submit(Job job)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
         JobIdData jobIdData = null;
         try {
+            job.setGenericInformation(DataUtility.setSubmissionModeToGenericInfo(job.getGlobalGenericInformation()));
```

Comment:
why externalize this method in DataUtility as it's apparently used only inside SchedulerClient (unless I missed one use) ?